src/net_processing.cpp

```diff
@@ -971,7 +980,7 @@ void PeerManagerImpl::MaybeSetPeerAsAnnouncingHeaderAndIDs(NodeId nodeid)
 {
     AssertLockHeld(cs_main);
 
-    // Never request high-bandwidth mode from peers if we're blocks-only. Our
+    // Never request high-bandwidth mode from peers in -blocksonly mode. Our
```

Comment:
This reads to me as the peer that's in -blocksonly mode. Maybe "When in -blocksonly mode, never request high-bandwidth mode from peers." ?

Reply:
Thx, done